### ISSUE TITLE
1.1.0 expected date?

### ISSUE THREAD

**Issue by andreassavva** [MacKentoch/react-native-beacons-manager#70]
This is not an issue, it's a question. Do we have an approximation on when version 1.1.0 is supposed to be released?

**Comment by cielocoderep** [comment]
yeah, examples are already in version 1.1.0 so its not working, when adding its adding v. 1.0.7
